x/swap/keeper/swap.go

```diff
@@ -103,7 +103,7 @@ func (k Keeper) commitSwap(
 	}
 
 	if err := k.supplyKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleAccountName, requester, sdk.NewCoins(swapOutput)); err != nil {
-		panic(err)
+		return err
```

Comment:
This error is a panic since the module account will always have funds equal or greater than the swap output amount.  If this error occurs, it's a bug and the module invariant will be broken, so a panic is preferred.  Normal circumstances will never cause this error, and there isn't a way to sign a message that will produce this error.